src/components/LayoutColumns/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
 import {map} from 'lodash'
 
-export default ({items}) => (
-  <div className='flex-ns'>
+export default ({items, relativeSizes}) => (
+  <div className='flex-l'>
     {map(items, (item, index) => (
       <div
         key={index}
-        className={`w-100-ns ${index < items.length - 1 ? 'mr4-ns' : ''}`}
+        className={`
+          ${index < items.length - 1 ? 'mr4-l' : ''}
+        `}
+        style={{
+          flex: relativeSizes
+            ? relativeSizes[index]
+            : 1
+        }}
```

Comment:
Thanks :) Props to @vojtaholik. More goodness coming soon: 
https://app.zeplin.io/project.html#pid=58caceae349a0d1341e322a5